cmd/web-handlers.go

```diff
@@ -700,12 +702,21 @@ func (web *webAPIHandlers) Upload(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Get host and port from Request.RemoteAddr.
+	host, port, err := net.SplitHostPort(handlers.GetSourceIP(r))
+	if err != nil {
+		host, port = "", ""
```

Comment:
should this be logged if host and port are not present?

Reply:
They are almost always present @kannappanr, its okay to ignore it IMO.